downstream/modules/platform/proc-enable-pac.adoc

```diff
@@ -0,0 +1,60 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-08
+:_mod-docs-content-type: PROCEDURE
+
+[id="enable-pac_{context}"]
+= Enabling policy enforcement
+
+During installation, you must configure your {PlatformNameShort} instance to include the policy enforcement feature. You can do this by modifying the feature flag variables in your configuration file.
+Follow the instructions below relevant to your installation type.
+
+== {OCPShort} Installation
```

Comment:
Procedures can not have subheadings. Either break this into procedure modules or format them in this file as follows:

.{OCPShort} Installation
<content>

.RPM Installation
<content>

.Containerized Installation
<content>